### PR TITLE
Feat/secrets management

### PR DESCRIPTION
**Pull request by rogefm**
Features:
- Hide routes of portal-web;
- Hide signing;
- Bug fixes;

### CODE REVIEW

**Comment by rogefm** [comment]
Hey Sandro, just remembering you about what we discussed before, this is a working version with agent UI running inside the agent (we've made a lot of fixes and now it's working) for our first goal: having a working version using memory to manage the secrets. If you could please approve it, we will move to the next step, which is having the UI communicate with the API, which will be focused on Andrios using it on sales calls.

File: Dockerfile [hunk 1]
```diff
@@ -182,7 +182,7 @@ ENV LC_ALL en_US.UTF-8
 
 ENV PATH="/opt/mssql-tools/bin:${PATH}"
 
-ADD target/uberjar/agent-$VERSION-standalone.jar /agent/app.jar
+ADD target/default+uberjar/agent-$VERSION-standalone.jar /agent/app.jar
```
Comment (sandromello) [inline Dockerfile:185]: You're going to break the build here

Comment (rogefm) [reply Dockerfile:185]: My bad

File: project.clj [hunk 2]
```diff
@@ -1,4 +1,4 @@
-(defproject agent "0.0.0"
```
Comment (sandromello) [inline project.clj:1]: Revert this change because the version is computed on build

Comment (rogefm) [reply project.clj]: Done

**Review by sandromello — APPROVED** [review]
